client.go

```diff
@@ -49,7 +59,7 @@ func (s *Session) Login(username string, email string, password string) error {
 	digest := sha256.Sum256([]byte(password))
 
 	loginResult, err := s.ddp.CallMethod("login", ddpLoginRequest{
-		User:     ddpUser{Email: email, Username: username},
```

Comment:
Die username is wel nodig. Niet elke client heeft een email adres. Beide velden hebben omitempty aan staatn, dus dat kan geen kwaad

Reply:
Fixed!